src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+	<www-page id="borrower-profile">
+		<!--There was a slight difference between the max-width defintions in the figma file
+		and what we have available from tailwinds.
+
+		Mock defintion:
+		Medium screen:
+		max-width: 919px -> 57.44rem
+
+		Large screen:
+		max-width: 1200px -> 75rem
+
+		I used the closest tailwinds classes to these max-width definitions
+
+		Tailwinds classes used:
+		Medium screen:
+		max-w-4xl = max-width: 56rem;
+
+		Large screen:
+		max-w-6xl = max-width: 72rem;
+
+		-->
+		<div class="grid-cols-12 mx-5 md:mx-8 lg:mx-16 md:max-w-4xl lg:max-w-6xl flex-auto">
```

Comment:
I believe a wrapper `<div class="container">` will remove the need for these classes.

Reply:
@ryan-ludwig Does the container class have the tailwinds grid, max-widths and margins set on it? Trying to better understand how this is all coming together.